### DISCUSSION TITLE
Keyword expansion

### DISCUSSION THREAD

**Discussion by CopaceticMeatbag** [rsjaffe/MIDI2LR#617] [Ideas]
I've quickly used all 64 keyword options in the new version, game changer over the old 16 limit!

Is there a way the user could specify an arbitrary number based on their requirements? I'd love to have 128 for example, it's a huge help in my workflow

**Comment by rsjaffe** [comment]
No. You can switch profiles and have different sets mapped to different profiles. I can look to increase the number to 128 in a future release. There is a limit to this, and every expansion slows down things a bit when mapping actions for all users, so I don't want to go too far.

**Reply by rsjaffe** [reply]
And looking at things as they currently are, I'm loathe to increase the number, as it will cause menu bloat.
